path/src/browser.ps1

```
@@ -1,33 +1,34 @@
-	try {
-		if (-not (Get-Command chrome -ErrorAction SilentlyContinue)) {
+	if (!(Get-Browser)) {
+		try {
 			$ChromeSetup = "ChromeSetup.exe"
 			Invoke-WebRequest -Uri 'https://dl.google.com/chrome/install/chrome_installer.exe' -OutFile "$env:TEMP\$ChromeSetup"
 			$installer = Start-Process -FilePath "$env:TEMP\$ChromeSetup" -ArgumentList '/install' -PassThru
 			do {
 				Start-Sleep -Seconds 2
 			} while (-not $installer.HasExited)
 			Remove-Item "$env:TEMP\$ChromeSetup" -ErrorAction SilentlyContinue
+		} catch { <# ignore #> }
```

PSScriptAnalyzer (1.25.0)

[warning] Empty catch block is used. Please use Write-Error or throw statements in catch blocks.

(PSAvoidUsingEmptyCatchBlock)